### PR TITLE
Update feature_column_v2_test.py

### PR DESCRIPTION
**Pull request by larrytin**

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Review by ymodak — APPROVED** [review]

**Comment by ymodak** [comment]
Thanks @larrytin  Can you please resolve the conflicts?

**Comment by rmlarsen** [comment]
@larrytin could you please resolve the conflict?

**Comment by rthadur** [comment]
@larrytin  gentle ping

File: tensorflow/python/feature_column/feature_column_v2_test.py [hunk 1]
```diff
@@ -3419,8 +3419,8 @@ def _initializer(shape, dtype, partition_info):
     self.assertEqual(1 + 3 + 5, net.shape[1])
     with _initialized_session() as sess:
 
-      # Each row is formed by concatenating `embedded_body_style`,
-      # `one_hot_body_style`, and `price` in order.
+      # Each row is formed by concatenating `one_hot_body_style`,
```
Comment (rohan100jain) [inline tensorflow/python/feature_column/feature_column_v2_test.py]: I think the original comment was fine. We sort the feature column names and this seems okay to me.

**Review by rohan100jain — CHANGES_REQUESTED** [review]

**Comment by rthadur** [comment]
closing this PR based on latest comments from reviewer.

**Pull request closed without merging**